scene/gui/split_container.h

```diff
@@ -62,13 +63,15 @@ class SplitContainer : public Container {
 	};
 
 private:
-	int split_offset = 0;
-	int middle_sep = 0;
+	Vector<int> split_offsets;
```

Comment:
```suggestion
	PackedInt32Array split_offsets;
```
I'd replace all usages of `Vector<int>` with this, it's clearer especially for exposed API.